src/Symfony/Component/VarDumper/Dumper/HtmlDumper.php

```diff
@@ -163,8 +163,8 @@ protected function getDumpHeader()
 <script>
 Sfdump = window.Sfdump || (function (doc) {
 
-if (doc.body instanceof HTMLElement) {
-    doc.body.classList.add('sf-js-enabled');
+if (doc.documentElement) {
```

Comment:
Can this value ever be falsy?

Reply:
Browsers usually "fix" it, however I don't know about all browser tweaks - that's why I added the check here.

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/Document/documentElement?retiredLocale=de#notes

> For any non-empty HTML document, documentElement will always be an [<html>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/html) element. For any non-empty XML document, documentElement will always be whatever element is the root element of the document.

Reply:
What does it have to do with Selenium?

Anyways if documentElement is falsy I don't think the following script will serve any purpose since the dump won't appear to work. Maybe it would then easier to debug if an error is logged in the console 🤔

Reply:
I've remove the check. @MatTheCat are you happy with this?

Reply:
LGTM, thanks 👍